script.js

```diff
@@ -0,0 +1,855 @@
+import Producto from "./Producto.js" 
+import Carrito from "./Carrito.js" 
+
+//DOM PARA EL PAGO_REALIZADO.HTML
+// funcion para modificar la informacion del pago
+// recupero los valores del sessionStorage
+let mdpPagoRealizado = sessionStorage.getItem('mdp')
```

Comment:
Usa `const` en lugar de `let` para `mdpPagoRealizado`, ya que no necesitas modificar el tipo de valor después. Si planeas cambiar su contenido posteriormente, está bien usar let, pero si solo necesitas declarar y no cambiar el tipo, let no es la mejor elección.
```suggestion
const mdpPagoRealizado = sessionStorage.getItem('mdp')
```

Reply:
Si, usaba let para cambiarle el valor numerico recibido por el option del select que seleccionaba el usuario. Ahora agregue una variable dentro `medioDePago` y esa se asigna al DOM con un `textContent = medioDePago` dependiendo la opcion del usuario.